src/App.js

```diff
@@ -1,14 +1,18 @@
 import React, { Component } from 'react';
-import SplashPage from './components/SplashPage';
 import GemsPage from './components/GemsPage';
+import MetalsPage from './components/MetalsPage';
+import ResultsPage from './components/ResultsPage';
```

Comment:
Try to avoid naming components with this 'Page' suffix -- remember, React allows you to create single page applications -- so each component is more like a *piece* of a page, rather than an entire/actual page. You won't see components named this way in the wild.